Update google-ads-event-streaming.md (#110)
amplitude/amplitude-dev-center: docs/data/destinations/google-ads-event-streaming.md

- You must enable this integration in each Amplitude project you want to use it in.
- Amplitude sends custom events using Amplitude `event_type` as event name.

> - This integration uses the [Upload Click Conversions API](https://developers.google.com/google-ads/api/docs/conversions/upload-clicks)).

Reviewer: ```suggestion
- This integration uses Google's [Upload Click Conversions API](https://developers.google.com/google-ads/api/docs/conversions/upload-clicks)).
```

> - A user property **Google Click ID (GLID** or **initial_gclid** is required in order for Amplitude to create a click conversion and pass that event to Google Ads. A Google Click ID (GCLID) is a parameter passed in the URL with ad clicks, to identify the campaign and other attributes of the click associated with the ad for ad tracking and campaign attribution. In Google Ads, this is enabled by turning on the auto-tagging setting. Google Ads relies on the gclid (Google Click ID) parameter to track user interactions with ads and attribute conversions back to specific ad clicks. When a user clicks on a Google Ads ad, a unique gclid value is generated and appended to the URL. This gclid value is used to identify the click and link it to subsequent conversions.

Reviewer: ```suggestion
- This integration requires a Google Click ID (`gclid` or `initial_gclid` to enable Amplitude to create the click conversion and pass that event to Google Ads. Google Ads relies on the `gclid` parameter to track user interactions with ads and attribute conversions back to specific ad clicks. When a user clicks an ad served through Google Ads, Google Ads generates a unique `gclid` value and appends it to the URL. This parameter identifies the click, and links it to subsequent conversions.
```

#### Configure settings

1. On the **Settings** tab, click **Edit**.

> 2. Enter your [**Google Ads Customer ID**](https://support.google.com/google-ads/answer/1704344?hl=en). A Google Ads Customer ID is a unique identifier assigned to each advertiser or business that uses Google Ads, which is Google's online advertising platform. This ID helps Google track and manage accounts, campaigns, and billing information for advertisers. It is essential for accessing and managing your Google Ads account.

Reviewer: ```suggestion
2. Enter your [**Google Ads Customer ID**](https://support.google.com/google-ads/answer/1704344?hl=en). A Google Ads Customer ID is a unique identifier assigned to each advertiser or business that uses Google Ads. This ID helps Google track and manage accounts, campaigns, and billing information for advertisers.
```

> 3. Enter your [**Google Ads Conversion Action ID**](https://support.google.com/google-ads/thread/105330243?hl=en&sjid=5504033552721490234-EU). A Google Ads Conversion Action ID is a unique identifier associated with a specific conversion action within your Google Ads account. In Google Ads, a conversion action represents a desired action that you want visitors to your website to take, such as making a purchase, filling out a contact form, or signing up for a newsletter. The Conversion Action ID helps you track and measure the success of these actions and their associated campaigns.

Reviewer: ```suggestion
3. Enter your [**Google Ads Conversion Action ID**](https://support.google.com/google-ads/thread/105330243?hl=en&sjid=5504033552721490234-EU). This is a unique identifier associated with a specific conversion action in Google Ads, and represents a desired action that you want your visitors or users to take. For example, make a purchase, submit a contact form, or sign up for a newsletter.
```

> 4. Enter your [**Base64 encoded Google Cloud Service Account**](https://developers.google.com/google-ads/api/docs/oauth/service-accounts). A Google Cloud Service Account, also known as a Service Account, is a special type of Google account that is used for server-to-server interactions and authentication within Google Cloud Platform (GCP) services. It's typically used for granting permissions and access to resources like Google Cloud Storage, Google Cloud Compute Engine, or other GCP services. Service Accounts have associated credentials (private keys or OAuth tokens) that are used for authentication when interacting with GCP services programmatically.

Reviewer: ```suggestion
4. Enter your [**Base64 encoded Google Cloud Service Account**](https://developers.google.com/google-ads/api/docs/oauth/service-accounts). A Google Cloud Service Account, also known as a Service Account, is a special type of Google account used for server-to-server interactions and authentication within Google Cloud Platform (GCP) services. It's can grant permissions and access to resources like Google Cloud Storage, Google Cloud Compute Engine, or other GCP services. Service Accounts have associated credentials (private keys or OAuth tokens) for authentication when you interact with GCP services programmatically.
```

2. Toggle **Status** from **Disabled** to **Enabled**.
3. Save when finished.

## Use Cases

1. **Accurate Conversion Tracking:** The integration allows you to track conversions more accurately by sending Amplitude event data directly to Google Ads. This ensures that you can attribute conversions to specific user actions and events, helping you understand which aspects of your campaigns are driving results.
2. **Optimizing Campaigns:** With real-time data from Amplitude, you can optimize your Google Ads campaigns based on user behavior and engagement. This means you can make data-driven decisions to allocate your budget to the most effective keywords, ad creatives, and targeting options.

> 3. **Personalization:** By leveraging Amplitude's event data, you can create highly personalized ad campaigns. For example, you can use the data to target users who have completed specific actions on your website or app with tailored ad messages or offers.

Reviewer: ```suggestion
3. **Personalization:** With Amplitude's event data, you can create highly personalized ad campaigns. For example, you can use the data to target users who have completed specific actions on your website or app with tailored ad messages or offers.
```
